android-json-form-wizard/src/main/java/com/vijay/jsonwizard/task/AttachRepeatingGroupTask.java

```diff
@@ -192,7 +192,7 @@ private void formatRepeatingGroupLabelText(EditText referenceEditText, TextView
 
     protected void addUniqueIdentifiers(JSONObject element, String uniqueId) throws JSONException {
         // make repeating group element key unique
-        String currKey = element.getString(KEY);
+        String currKey = widgetArgs.getJsonObject().getString(KEY) + "_" + element.getString(KEY);
```

Comment:
This change will break stuff for us.

Reply:
This is not absolutely necessary for this pr, so it can be removed if no other compromise is found.

Reply:
I think its best for other implementations to use the current key as is without the new prefix .

Reply:
Removed